### PR TITLE
support prune cmd

### PR DESCRIPTION
**Pull request by xiaozhouX**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/arena/100)

### CODE REVIEW

File: cmd/arena/commands/list.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func NewListCommand() *cobra.Command {
 			trainers := NewTrainers(client)
 			jobs := []TrainingJob{}
 			// for
-
+			println(name)
```
Comment (cheyang) [inline cmd/arena/commands/list.go]: Is this for debugging?

File: cmd/arena/commands/prune.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+				if GetJobRealStatus(job) != "RUNNING" {
+					if job.Age() > (time.Duration(pruneArgs.days)*24*time.Hour + time.Duration(pruneArgs.hours)*time.Hour) {
+						deleteTrainingJob(job.Name())
+					}
+				}
+			}
+		},
+	}
+
+	command.Flags().Int64Var(&pruneArgs.days, "day",  10, "Specify clean job's days.")
```
Comment (cheyang) [inline cmd/arena/commands/prune.go]: I think it's more Intuitive to use --period to specify the prune criteria.  Just like --period=10d or 10h.  Thanks. No urgent.

**Comment by cheyang** [comment]
Thanks very much for your awesome contribution!  Please check my comments. Thanks.

File: cmd/arena/commands/list.go (hunk 1 above)
Comment (xiaozhouX) [reply cmd/arena/commands/list.go]: Yes I will remove it

File: cmd/arena/commands/prune.go (hunk 2 above)
Comment (xiaozhouX) [reply cmd/arena/commands/prune.go]: Sound reasonable

File: cmd/arena/commands/prune.go [hunk 3]
```diff
@@ -79,17 +78,25 @@ func NewPruneCommand() *cobra.Command {
...
 			}
+			if !deleted {
+				fmt.Println("No job need to be deleted")
+			}
 		},
 	}
 
-	command.Flags().Int64Var(&pruneArgs.days, "day",  10, "Specify clean job's days.")
-	command.Flags().Int64Var(&pruneArgs.hours, "hour", 0, "Specify clean job's hours.")
+	command.Flags().DurationVarP(&pruneArgs.since, "since","s",  24 * time.Hour, "Specify clean job's days.")
```
Comment (cheyang) [inline cmd/arena/commands/prune.go]: 1. we won't do any prune if the user doesn't specify  the relative duration for cleaning up.
2. Need clarify usage info. I think it's better to say "Only prune the jobs created before the given relative duration like 5s,2m, 3h or 10d"

**Comment by cheyang** [comment]
please fix ci issue. Thanks.

**Pull request merged**